<a href="https://colab.research.google.com/github/marco-canas/algebra/blob/main/Clase_7_sep_2022.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

[Vídeo de apoyo a la lectura experimental e interactiva de este cuaderno]()

# Construcción de modelos en aplicaciones de las matemáticas a la economía  
En los ejemplos y ejercicios que siguen planteamos ecuacioes que modelan problemas en muchas situaciones distintas de la vida cotidiana.  
## Ejemplo 2 Interés de una inversión  
Leonardo hereda 450 millones de pesos y los invierte en dos certificados de depósito a término. Uno de los certificados paga el 6% y el otro paga $4.5$% de interés anual simple.   Si el interés total de Leonardo es $22'491.900 pesos por año, ¿cuánto dinero está invertido en cada tasa?  
1) Mire la pregunta y de ahí, defina la incógnita  

##Conceptos asociados al problema  
* La propiedad distributiva de la multiplicación con respecto a la suma. 
* Interés simple e interés compuesto.  
* Propiedad de la balaza o propiedad uniforme.  
* Transponer términos y factores.

X: lo invertido al 6  

450 - X es lo invertido al 4,5$%   

Modelo:

Lo invertido al 6% + Lo invetido al 4,5% = 450 Millones (Total)  

X + (450 - X) = 450  

Interés de lo invertido al 6% + Interés de lo invertido al 4.5% = Inversión total (22,491)    
## El modelo de esta situación económica es:

$$X(0,06) + (450 . X) (0,045)= 22,491$$ interés total 



In [5]:
from sympy import *

x = var('x')
modelo = Eq(x*0.06+ (450-x)*0.045, 22.4919)
solve(modelo, x)[0]

149.460000000000

In [6]:
450-solve(modelo, x)[0]

300.540000000000

Lo invertido al 6% efectivo anual es de $149'460.000 y lo invetido al 4.5% es $$300'540.000 y me parece muy desascertado lo que hizo Leonardo pues es mejor invertir más a una tasa más alta de interés.

In [7]:
300*0.06

18.0

In [8]:
150*0.045

6.75

In [9]:
300*0.06+150*0.045

24.75

Note que hubiera ganado casi 25 millones de interés si hubiera invertido más dinero a una tasa más alta.